apps/webservice/src/app/[workspaceSlug]/(app)/(deploy)/(raw)/systems/[systemSlug]/(raw)/environments/[environmentId]/resources/ResourcesPageContent.tsx

```
@@ -0,0 +1,570 @@
+          </div>
+        </div>
+
+        <div className="rounded-lg border border-neutral-800 bg-neutral-900/50 p-4">
+          <div className="mb-1 flex items-center gap-1.5 text-xs text-neutral-400">
+            <div className="h-2 w-2 rounded-full bg-amber-500"></div>
+            <span>Needs Attention</span>
+          </div>
+          <div className="text-2xl font-semibold text-amber-400">{0}</div>
+          <div className="mt-1 flex items-center text-xs">
+            <span className="text-amber-400">
+              {0 > 0 ? "Action required" : "No issues detected"}
```

Biome (1.9.4)

[error] Comparing to itself is potentially pointless.



(lint/suspicious/noSelfCompare)